integration/single-node-slug-deploy/check.go

```diff
@@ -441,13 +441,13 @@ func verifyTransferReplicas(errCh chan<- error, tempdir string, logger logging.L
 	builder.SetID("some_pod")
 	man := builder.GetManifest()
 
-	fromRC, err := rcStore.Create(man, klabels.Everything(), "some_az", "some_cn", nil, nil)
+	fromRC, err := rcStore.Create(man, klabels.Everything(), "some_az", "some_cn", nil, nil, "some_strategy")
 	if err != nil {
 		errCh <- util.Errorf("could not create RC for replica transfer test: %s", err)
 		return
 	}
 
-	toRC, err := rcStore.Create(man, klabels.Everything(), "some_az", "some_cn", nil, nil)
```

Comment:
mental note to myself that we'll probably have to make this a real strategy once the RC farm actually looks at this (the integration test runs a real RC farm!)

Reply:
I thought about putting `PetStrategy` in instead of `"some_strategy"` but didn't know if it was overkill. I no longer check if a string is a valid strategy by doing a map lookup so we may not need to make it an actual strategy if we just let it fall through to the default case/branch